### PR TITLE
Loop variable names

### PR DESCRIPTION
**Pull request by sparce**
Changes loop variable names to be more meaningful per issue #725 

Think I caught them all except for the one in the [functions episode](http://swcarpentry.github.io/python-novice-inflammation/06-func/index.html#readable-functions) that is supposed to be hard to interpret. 

(First time attempting a PR for the lessons, so feel free to let me know if there's anything I should be doing differently for next time)

### CODE REVIEW

**Comment by sparce** [comment]
I should also add that I noticed some inconsistencies between the code in the lesson notes for [command line programs](http://swcarpentry.github.io/python-novice-inflammation/10-cmdline/index.html) and the actual scripts in the `code` directory.

For example, the lesson notes show the `readings_*.py` files using `numpy.mean(data, axis=1)`, while the scripts themselves use `data.mean(axis=1)`. 
Some of the solutions in the notes (eg. (changing flags)[http://swcarpentry.github.io/python-novice-inflammation/10-cmdline/index.html#changing-flags]) also just call `main()` directly at the end of the script, whereas all the script files check `if __name__ == '__main__':` first.

These differences don't matter for how the scripts run, but it's probably worth keeping the notes consistent with the scripts that learners will be downloading. Would it be better to start a separate issue for this, or should I just add corrections to this PR since I'm already working with the files in the code directory?

**Comment by ldko** [comment]
@sparce thanks for noting the inconsistencies between the lessons and `code` directory. I agree with you that it is best to keep the sample code and instruction consistent. Changes for that should go wit a separate issue and PR. Thank you!

File: _episodes/04-files.md [hunk 1]
```diff
@@ -162,11 +162,11 @@ where the maxima are a bit less regular, but the minima are consistently zero.
...
 > > composite_data = numpy.zeros((60,40))
 > >
-> > for f in filenames:
-> >     data = numpy.loadtxt(fname = f, delimiter=',')
+> > for filename in filenames:
+> >     data = numpy.loadtxt(fname = filename, delimiter=',')
 > >     composite_data += data
 > >
-> > composite_data/=len(filenames)
+> > composite_data /= len(filenames)
```
Comment (maxim-belkin) [inline _episodes/04-files.md]: @ldko and @annefou, we need to check that we mention compound operators in the introduction

**Comment by maxim-belkin** [comment]
Thank you, Stephen.
Are you comfortable splitting the PR into several smaller ones, updating one file per PR?

File: _episodes/04-files.md (hunk 1 above)
Comment (annefou) [reply _episodes/04-files.md]: We do not mention compound operators in the introduction. But we do it here. To be honest, I sometimes do not mention it and use
composite_data = composite_data + data
etc.
Maybe we should add a note in the instructor guide; what do you think?

**Comment by sparce** [comment]
@maxim-belkin Is there a way to break it up from here, or would I need to start again with a separate branch and PR for each episode? I'm happy to handle it either way.

**Comment by maxim-belkin** [comment]
I see that you're adept at using GitHub, so may I ask you to follow the guidelines in our Wiki: https://github.com/swcarpentry/python-novice-inflammation/wiki/Splitting-(Breaking-down)-Pull-Requests and let me know if the instructions there work for you?

File: _episodes/04-files.md (hunk 1 above)
Comment (maxim-belkin) [reply _episodes/04-files.md]: @nspies, please ignore this discussion when doing the PR split -- we'll address this in a separate PR. You changes are good.

**Comment by maxim-belkin** [comment]
Thank you for splitting this PR into multiple ones, Stephen!
Working with small "limited-in-scope" commits is a lot easier and usually results in faster turn-around times.

**Pull request closed without merging**